src/App.js

```diff
@@ -0,0 +1,21 @@
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import React from 'react';
+import NavBar from './components/navBar/NavBar';
+import Books from './components/Books/Books';
+import Categories from './components/Categories/Categories';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <NavBar />
+      <main>
+        <Routes>
+          <Route path="/books" element={<Books />} />
```

Comment:
- [ ] In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page. So let's add the `exact` prop to the route and change the `path` value to `"/"` to handle that.

Reply:
@EricMbouwe Thank you so much for your review but 
could you explain more what you mean by "So let's add the exact prop to the route" ?

Reply:
`exact` is a route prop that create the exact matching between the url route entered by the user, the path you defined in your route tag and the component you want your app to render when that url is called.

[more info about](https://www.oreilly.com/library/view/react-router-quick/9781789532555/13c3acc3-c0a2-48fc-a0a6-e3a780e6cad7.xhtml)